### DISCUSSION TITLE
Granting accessibility permissions for XCUITest on GitHub-hosted runner

### DISCUSSION THREAD

**Discussion by p0deje** [community/community#39846] [Actions]
### Select Topic Area

Bug

### Body

I suppose I need to do something extra to run Xcode UI Tests on GitHub Actions in my open-source project https://github.com/p0deje/Maccy. Whenever those run, the popup shows asking for accessibility permissions to run the tests and the tests simply fail with timeouts.

![screenshot](https://user-images.githubusercontent.com/665846/203578791-aab03c35-1620-4069-87ad-93dfeb978bd0.jpg)

My workflow is as simple as https://github.com/p0deje/Maccy/actions/runs/3532897919/workflow

Has anyone solved this?

**Comment by david-engelmann** [comment]
It seems like you need to update your Xcode!

**Reply by p0deje** [reply]
Why do you think so? It uses Xcode 14.0.1 which is the default version installed on https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md.
